integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/projection/SearchProjectionIT.java

```diff
@@ -608,6 +609,7 @@ private void initData() {
 
 		final IndexObjectFieldReference nestedObject;
 		final FieldModel<String> nestedField;
+		final FieldModel<GeoPoint> nestedGeoPoint;
```

Comment:
It looks like you're not doing anything with that new field?

Reply:
That's right! I would have used it to test nested distance projection, but then I created `NestedDocumentDistanceProjectionIT.java` for that.